### PR TITLE
Add guidelines for contribution

### PR DESCRIPTION
**Pull request by iranzo**
Adapt kube community guidelines/readme to metal3

### CODE REVIEW

File: GUIDELINES.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
+# Contents Guidelines
+
+This document describes a set of guidelines for generating contents for [metal3.io](https://metal3.io ), exceptions can be make if and when it makes sense, but please try to follow this guide as much as possible.
```
Comment (matthewcarleton) [inline GUIDELINES.md]:
```suggestion
This document describes a set of guidelines for generating contents for [metal3.io](https://metal3.io ), exceptions can be made if and when it makes sense, but please try to follow this guide as much as possible.
```

File: GUIDELINES.md [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+* Follow [Kramdown Quick Reference](https://kramdown.gettalong.org/quickref.html) for syntax reference
+* Split the contents in sections using the different levels of headers that Markdown offers
+  * Keep in mind that once rendered, the title you set in the Front Matter data will use *H1*, so start your sections from *H2*
+* Closing section, the same way we can add a brief opening section describing what the contents is about, it's very important to add a closing section with thoughts, upcoming work on the topic discussed, encourage readers to test something and share their findings/thoughts, joining the community, ... keep in mind that this will probably be the last thing the reader will read
+* [Code blocks](https://kramdown.gettalong.org/syntax.html#code-blocks), use them for:
+  * code snippets
+  * file contents
+  * console commands
+  * ...
+  * Use the proper tag to let the renderer what type of contents your including in the block for syntax highlighting
```
Comment (matthewcarleton) [inline GUIDELINES.md]:
```suggestion
  * Use the proper tag to let the renderer know what type of contents your including in the block for syntax highlighting
```

File: GUIDELINES.md [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+  ```
+
+* The fields have the same function as for blog posts, but some values are different, as we're producing different contents.
+  * **permalink** tells *Jekyll* what the output path for your page will be, it's useful for linking and web indexers
+  * **navbar_active** will add your page to the navigation bar you specify as value, commonly used values are *Docs* or *Videos*
+  * **layout**, just use *default* as value, it'll include all the necessary parts when your page is generated
+
+* As for the contents, follow the general guidelines above
+
+### Labs
```
Comment (matthewcarleton) [inline GUIDELINES.md]: Have we confirmed this site will contain labs?

Comment (iranzo) [reply GUIDELINES.md]: From the spreadsheet we've of possible content, labs are listed ```Blog posts to troubleshoot rhhi.next (Issue vs resolution labs?)```

**Comment by iranzo** [comment]
@matthewcarleton can we get this merged before moving to final repo?

Thanks,
Pablo

**Pull request merged**